server/config/permissions.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
 
         'specialaccount:read',
         'specialaccount:write',
-        'specialaccount:force-code-reset'
+        'specialaccount:force-code-reset',
+
```

Comment:
Empty line ?

Reply:
Yep, it's wanted (to distinguish groups)